bot/resources/tags/blocking.md

```diff
@@ -0,0 +1,22 @@
+**What is asynchronous programming?**
+
+An asynchronous program doesn't wait for step to be finished before executing another one, but just keeps continuing executing other steps.
+
+It will also know what to do when the previous step finishes executing.
+
+Consider this example from [Miguel Grinberg’s 2017 PyCon talk](https://www.youtube.com/watch?t=4m29s&v=iG6fr81xHKA&feature=youtu.be), about playing multiple games of chess at once.
+
+
+**What does blocking mean?**
+
+In asynchronous programming, blocking calls are all the parts of your function that are not using `await`.
```

Comment:
> In asynchronous programming, blocking calls are all the parts of your function that are not using `await`.
> 
> Not all forms of blocking are bad, and using blocking calls are inevitable, but make sure not to use too much.

I think this part would be a bit vague for someone who's new to async/await. `x + y` is a blocking call, but you wouldn't usually describe it as such. Maybe clarify that it applies mostly to blocking I/O (with examples such as `open` and `requests`), or to CPU-heavy tasks.

Reply:
OK